extmod/modframebuf.c

```diff
@@ -319,11 +336,8 @@ STATIC void framebuf_args(const mp_obj_t *args_in, mp_int_t *args_out, int n) {
 }
 
 STATIC mp_int_t framebuf_get_buffer(mp_obj_t self_in, mp_buffer_info_t *bufinfo, mp_uint_t flags) {
-    (void)flags;
     mp_obj_framebuf_t *self = MP_OBJ_TO_PTR(self_in);
-    bufinfo->buf = self->buf;
-    bufinfo->len = self->stride * self->height * (self->format == FRAMEBUF_RGB565 ? 2 : 1);
-    bufinfo->typecode = 'B'; // view framebuf as bytes
+    mp_get_buffer_raise(self->buf_obj, bufinfo, flags);
```

Comment:
This should return 1 if it doesn't support the buffer protocol, not raise an exception.  So maybe calling `mp_get_buffer()` is conceptually better here (even though it's never going to raise because the buf is already validated to be a buffer in the FrameBuffer constructor).

Reply:
Yep, fixed.